extensions/amp-orientation-observer/validator-amp-orientation-observer.protoascii

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-orientation-observer
+  html_format: AMP
```

Comment:
The script tag is allowed for AMP but the usage of the extension is allowed for AMP, AMP4ADS and EXPERIMENTAL. The set of these should match.